### PR TITLE
Issue-49 Give the DOCKER_CONTAINER env to compilation

### PR DESCRIPTION
**Pull request by aduprat**

### CODE REVIEW

**Comment by rouazana** [comment]
:+1:

File: workflow-job [hunk 1]
```diff
@@ -257,7 +257,7 @@ flows["${sha1}"] = {
         node {
             parallel("Building Java 8 ${buildId}": {
                 sh "docker build --tag=${images.jamesCompile} dockerfiles/compilation/java-8"
-                sh "cd dockerfiles/; docker run ${verbose} --name=${containers.jamesCompile} ${dockerEnv} ${useMavenCache} --volumes-from=${containers.keys} --volumes-from=${containers.gitPublish} --volume=/destination ${images.jamesCompile} ${mergeBranch}"
+                sh "cd dockerfiles/; docker run ${verbose} --name=${containers.jamesCompile} ${dockerEnv} ${useMavenCache} -e DOCKER_CONTAINER=${containers.jamesCompile} --volumes-from=${containers.keys} --volumes-from=${containers.gitPublish} --volume=/destination ${images.jamesCompile} ${mergeBranch}"
```
Comment (mbaechler) [inline workflow-job:260]: That's only for java 8 build ?

Comment (aduprat) [inline workflow-job:260]: Yes, we have no docker tests in Java 6

**Comment by mbaechler** [comment]
👍

**Pull request merged**